.node-version

```diff
@@ -1 +0,0 @@
-14.15.5
```

Comment:
Is it possible to also  use `>=14.0.0` here?

Reply:
I would prefer to revert this delete. Having this file here makes `nodenv` do the Right Thing when you `cd` into the directory. Just leave the README and `package.json` looser, and have some version opinion in `.node-version` and we'll have the best of both worlds.

Reply:
That sounds reasonable to me